packages/jellyfish-api-core/src/category/net.ts

```diff
@@ -29,6 +38,38 @@ export class Net {
   }
 }
 
+export interface PeerInfo {
+  id: number
+  addr: string
+  addrbind: string
+  addrlocal: string
```

Comment:
```suggestion
  addrbind?: string
  addrlocal?: string
```
yea.. those contains condition.. just make it optional

Reply:
below has few condition properties which are required to set optional as well :)

Reply:
@canonbrother Would this also be the case for the following value assignments?

[1](https://github.com/DeFiCh/ain/blob/master/src/rpc/net.cpp#L157-L162)
[2](https://github.com/DeFiCh/ain/blob/master/src/rpc/net.cpp#L171-L180)

Reply:
yep

Reply:
Ok great, will update interface and tests accordingly